src/main/java/ch/njol/skript/expressions/ExprOnlinePlayersCount.java

```diff
@@ -51,8 +51,8 @@
 
 	static {
 		Skript.registerExpression(ExprOnlinePlayersCount.class, Number.class, ExpressionType.PROPERTY,
-				"[the] [(1¦(real|default)|2¦(fake|shown|displayed))] [online] player (count|amount|number)",
-				"[the] [(1¦(real|default)|2¦(fake|shown|displayed))] (count|amount|number|size) of online players");
+				"[the] [(1¦(real|default)|2¦(fake|shown|displayed))] [online] players (count|amount|number)",
```

Comment:
s should be made optional here, no need to create a breaking change IMO

Reply:
I agree however, this should be the correct syntax because `player amount` is just wrong don't you think?

Reply:
Discussed on discord.